air/src/aux_table/bitwise/tests.rs

```diff
@@ -75,6 +74,45 @@ fn test_bitwise_selectors_fail() {
     assert_ne!(result, expected);
 }
 
+#[test]
+/// Tests that the bitwise constraints do not all evaluate to zero if the operation selectors change
+/// within a cycle even when the output column will be the same for both of them.
+fn test_bitwise_pow2_selectors_fail() {
```

Comment:
The name of this test should be updated.

Reply:
Done! Please see [abb346a](https://github.com/maticnetwork/miden/pull/289/commits/abb346aa50875a16ebd6eb30f793475f20faf9ff).